### PR TITLE
Remove Gulp

### PR DESCRIPTION
**Pull request by andy128k**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -160,43 +158,6 @@ You can programmatically access Favicons configuration (icon filenames, HTML, ma
...
-        logging: false,
-        html: "index.html",
-        pipeHTML: true,
-        replace: true,
-      })
-    )
-    .on("error", log)
-    .pipe(gulp.dest("./"));
-});
-```
```
Comment (alexander-akait) [inline readme.md:198]: Maybe we can provide alternative example how to use? But I agree, gulp is very rare today

Comment (andy128k) [reply readme.md:198]: Such example would require importing `vinyl` package from `gulp`. And will not be as simple as current one.
This package will not be "gulp friendly" anymore. For those who need `gulp` there is still `gulp-plugin` available.

Comment (alexander-akait) [reply readme.md:198]: Sounds good

**Review by alexander-akait — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -45,8 +44,6 @@
     "escape-html": "^1.0.3",
     "lodash.defaultsdeep": "^4.6.1",
```
Comment (alexander-akait) [inline package.json:45]: I think `lodash.defaultsdeep` can be dropped too in favor `{ ...options }` syntax

Comment (andy128k) [reply package.json:45]: There are a few of nested fields, so it is not that straightforward.

Comment (alexander-akait) [reply package.json:45]: We can do nested clone, i.e. `{ foo: { ...options.foo } }`

**Pull request merged**